POEApi.Model.Tests/TestData/SampleCharacter.json

```diff
@@ -31,5 +31,6 @@
 	"league": "Standard",
 	"classId": 3,
 	"class": "Witch",
-	"level": 79
+  "level": 79,
+  "expired": true
```

Comment:
Instead of modifying the existing SampleCharacter.json. Please submit an SampleCharacterExpired.json for one of the characters you are experiencing this expiry issue.

Please add a supporting unit test for the expired property.